### PR TITLE
[2078] Add multiple users param on LuUserDisplayPipe

### PR DESCRIPTION
**Pull request by g-drouard**
## Description

Add multiple users param on LuUserDisplayPipe

N.B. Usage with single user param remains the same

-----

Examples : 
- In template: `{{ users | luUserDisplay: { format: 'LF', separator: '; '  } }}`
- With transform: `luUserDisplayPipe.transform(users, { format: 'LF', separator: '; ' });`

Defaut values:
- Format (same as single user): InjectionToken `LU_DEFAULT_DISPLAY_POLICY`
- Separator: `, `

Breaking changes :
- Before: Passing a null user displays an empty string
- After: Passing a null user throws an error

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/4745/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2088/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/4750/ui-diff)

File: tsconfig.json [hunk 1]
```diff
@@ -2,40 +2,31 @@
...
+			"@/stories/*": ["stories/documentation/*"]
+		},
+		"declaration": false,
+		"downlevelIteration": true,
+		"experimentalDecorators": true,
+		"moduleResolution": "node",
+		"importHelpers": true,
+		"target": "ES2022",
+		"module": "ESNext",
+		"lib": ["es2022", "dom"],
```
Comment (g-drouard) [inline tsconfig.json:27]: Intl is only supported with the es2022

**Review by F2X — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2088/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18368/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2088/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18372/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2088/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18377/ui-diff)

**Review by Supamiu — APPROVED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**